assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/CreateGoalCTA.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * CreateGoalCTA component.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { CORE_SITE } from '../../../../../googlesitekit/datastore/site/constants';
+import Button from '../../../../../components/Button';
+import PreviewGraph from './PreviewGraph';
+import GoalsGraph from '../../../../../../svg/graphics/cta-graph-goals.svg';
+const { useSelect } = Data;
+
+export default function CreateGoalCTA() {
+	const supportURL = useSelect( ( select ) =>
+		select( CORE_SITE ).getGoogleSupportURL( {
+			path: '/analytics/answer/1032415',
+			hash: 'create_or_edit_goals',
+		} )
+	);
+
+	return (
+		<div className="googlesitekit-cta--analytics googlesitekit-cta--analytics--half">
```

Comment:
Thanks, please see my reply to the related comment, [here](https://github.com/google/site-kit-wp/pull/4862/files#r811959523).

Reply:
Sounds good! Thanks @techanvil . See my [comment here](https://github.com/google/site-kit-wp/pull/4862#discussion_r812883813). Basically, once again we are limited since we need to be able to set the width of the cells at a higher breakpoint 😞